doc/how-to/manage-infrastructure.md

```diff
@@ -0,0 +1,119 @@
+# Manage Infrastructure
+
+The service is hosted on the [MoJ Cloud
+Platform](https://user-guide.cloud-platform.service.justice.gov.uk/#getting-started).
+It's a platform where we can host our applications and interact with them
+through Kubernetes. This requires our applications to all be Dockerised.
+
+Each repository will have a `helm_deploy` directory that specifies configuration
+for this service in each of the environments.
+
+When this app deploys via merging to `main`, CircleCI will automatically
+propagate those changes to the cluster. This happens through the
+`hmpps/deploy_env` job that is provided by the [MoJ CircleCI
+Orb](https://github.com/ministryofjustice/hmpps-circleci-orb).
+
+Each environment will correspond to a Cloud Platform 'namespace'. The namespace
+is an isolated cluster. We can use the [Cloud Platform Environments
+repository](https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live.cloud-platform.service.justice.gov.uk)
+to define our backing services, certificates etc.
+
+## Prerequisites
+
+- Be a part of the Ministry of Justice GitHub organisation. This should be done
+  as part of your your team onboarding process.
+- Be a part of the `accredited-programmes-team` team. This should be done as
+  part of your team onboarding process.
+- [Follow the Cloud Platform guidance to connect to the Kubernetes
+  cluster](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html#connecting-to-the-cloud-platform-39-s-kubernetes-cluster)
+
+## Kubernetes cheat sheet
+
+To use Kubernetes to interact with the cluster there's [a cheat
+sheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/). Keep reading
+for tasks we commonly use.
+
+## Common Kubernetes tasks
+
+### View the application logs of a pod
```

Comment:
Is a pod different to a cluster?

Reply:
Yes - [this documentation](https://kubernetes.io/docs/concepts/overview/components/) might be a helpful overview.

TL;DR a "cluster" is the thing that's deployed, which is a collection of "nodes" (worker machines) which host "pods" that run dockerised applications e.g. our application & the API in this case. So when we're debugging, we want to look at the output of these pods as they're what our app is running in.